test-requirements.txt

```diff
@@ -1,6 +1,10 @@
 -r requirements.txt
 
 mock >= 5.1.0, < 6
+autoflake==2.3.0
```

Comment:
These are usually installed directly in the Generator's Makefile, but since we'll also be running tests here, let's transition them into the development dependencies manifest directly.